elastic_enterprise_search/client/_app_search.py

```diff
@@ -1395,12 +1395,6 @@ def create_synonym_set(
             if param in SKIP_IN_PATH:
                 raise ValueError("Empty value passed for a required argument")
 
-        params = make_params(
```

Comment:
Everything under `client/` is auto-generated from specs, so if this is failing then this is likely a specification problem. I'll investigate.